Administrator Guide/setting-up-storage.md

```diff
@@ -0,0 +1,9 @@
+# Setting Up Storage
+
+A volume is a logical collection of bricks where each brick is an export directory on a server in the trusted storage pool.
+Before creating a volume, you need to set up the bricks that will form the volume.
+
+
+ -  [Brick Naming Conventions](./Brick Naming Conventions.md)
+ -  [Formatting and Mounting Bricks](./formatting-and-mounting-bricks.md)
+ -  [Posix ACLS](./Access Control Lists.md)
```

Comment:
Same comment here about ACLs.

Reply:
The page actually talks abt mounting the brick with -o acl .